### PR TITLE
Add error handling to inputs for edit profile

### PR DESCRIPTION
**Pull request by pernstrong**
#### What's this PR do?
Adds some error handling to the user input to edit profile.
#### Where should the reviewer start?
EditProfile.js
#### How should this be manually tested?
Try editing various fields in EditProfile.js
#### Any background context you want to provide?
n/a
#### What are the relevant tickets?
n/a
#### Screenshots (if appropriate)
n/a
#### Questions:
n/a

### CODE REVIEW

File: code/web/src/modules/user/EditProfile.js [hunk 1]
```diff
@@ -1,25 +1,26 @@
...
 
-    const [ name, updateName ] = useState()
-    const [ email, updateEmail ] = useState()
-    const [ shippingAddress, updateShippingAddress ] = useState(
-    )
-    const [ description, updateDescription ] = useState()
+    const [ name, updateName ] = useState('')
+    const [ email, updateEmail ] = useState('')
+    const [ shippingAddress, updateShippingAddress ] = useState('')
+    const [ description, updateDescription ] = useState('')
```
Comment (DavidTTran) [inline code/web/src/modules/user/EditProfile.js:23]: I trust you that hooks are a more efficient way to track states.

File: code/web/src/modules/user/Profile.js [hunk 2]
```diff
@@ -30,12 +30,23 @@ const Profile = (props) => (
...
         <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
 
+        {/* <Grid> */}
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+        <h3>Shipping Address:</h3>
+        <p>{props.user.details.shippingAddress}</p>
+        <br />
+        <h3>Description:</h3>
+        <p>{props.user.details.description}</p>
+      </GridCell>
```
Comment (raymondknguyen) [inline code/web/src/modules/user/Profile.js:48]: This looks great! figuring out this format must be difficult

**Review by nickstaylor — APPROVED** [review]
Added Error handling for Edit Profile Page

**Review by raymondknguyen — APPROVED** [review]

File: code/web/src/modules/user/EditProfile.js [hunk 3]
```diff
@@ -95,47 +87,73 @@ const EditProfile = (props) => {
...
+            <Grid alignCenter={true}>
+                <GridCell>
+                    <label for="name"  style={{ padding: '2em', margin: '5%', textAlign: 'center' }}>Name:</label>
+                    <Input style={{ padding: '2em', marginTop: '5%', textAlign: 'center' }}
+                            type="text"
+                            name="name"
+                            value={name}
+                            placeholder={props.user.details.name}
+                            onChange={e => updateName(e.target.value)}
+                        />
```
Comment (DavidTTran) [inline code/web/src/modules/user/EditProfile.js:99]: Odd this is just hanging out in the middle of nowhere

File: code/web/src/modules/user/api/actions.js [hunk 4]
```diff
@@ -137,6 +137,6 @@ export function updateProfile(userDetails) {
   }
   
   const setLocalStorageOnEditProfile = (user) => {
-    const userObj = {email: user.email, name: user.name, role: 'user', shippingAddress: user.shippingAddress}
+    const userObj = {email: user.email, name: user.name, role: 'user', shippingAddress: user.shippingAddress, description: user.description}
```
Comment (DavidTTran) [inline code/web/src/modules/user/api/actions.js:140]: So next step would be to include user.image

**Pull request merged**